src/SFA.DAS.ApplyService.Web/Services/OverallOutcomeService.cs

```diff
@@ -123,7 +126,11 @@ public async Task<ApplicationSummaryWithModeratorDetailsViewModel> BuildApplicat
             return model;
         }
 
-
+        public async Task<OversightReviewStatus?> GetOversightReviewStatus(Guid applicationId)
```

Comment:
Why would this be nullable? Isn't that what the `None` entry is for?

Reply:
taking this out